core/src/main/scala/chisel3/Bits.scala

```diff
@@ -177,6 +175,12 @@ sealed abstract class Bits(private[chisel3] val width: Width) extends Element wi
     }
   }
 
+  final def apply(x: Int, y: Int): UInt = macro SourceInfoTransform.xyArg
```

Comment:
We may end up not adding `extract`, but if we do, make sure not to get rid of ScalaDoc. All public methods need ScalaDoc.